src/JuAFEM.jl

```diff
@@ -14,10 +14,11 @@ import WriteVTK: vtk_grid
 
 export start_assemble, assemble, end_assemble
 
-export FEValues, reinit!, shape_value, shape_gradient, shape_divergence, detJdV, get_quadrule, get_functionspace,
+export FEValues, reinit!, shape_value, shape_gradient, shape_divergence, detJdV, get_quadrule, get_functionspace, get_geometricspace,
                  function_scalar_value, function_vector_value, function_scalar_gradient,
                  function_vector_gradient, function_vector_divergence, function_vector_symmetric_gradient
-export n_dim, n_basefunctions
+export FEFaceValues
+export FunctionSpace, n_dim, ref_shape, fs_order, n_basefunctions
```

Comment:
name suggestion? its just that the [old version](https://github.com/KristofferC/JuAFEM.jl/blob/master/src/function_spaces.jl#L29) didn't work

```jl
julia> JuAFEM.order
order (generic function with 0 methods)
```